tcwebhooks-core/src/main/java/webhook/teamcity/settings/WebHookConfig.java

```diff
@@ -592,6 +596,10 @@ public String getStateBuildBrokenAsChecked() {
 		return getAsChecked(BUILD_BROKEN);
 	}
 
+	public String getStateBbuildPromotionTagsChangedAsChecked() {
```

Comment:
```suggestion
	public String getStateBuildTagsChangedAsChecked() {
```
BTW, I am a little embarrassed that these "as checked" methods are still in here. They are a view specific details that the service layer shouldn't need to provide. I should've coded them in the JSP, not here. 

Some day soon this code will all go, and be replaced with calls to the REST API to CRUD webhooks. I am working on that on another branch.